solutions/lesson6.js

```diff
@@ -0,0 +1,10 @@
+const biggestValue = (obj,i=0,objArr=Object.values(obj),biggest=objArr[0])=>{
+  if(i==objArr.length){
+    return biggest;
+  }
+  if(objArr[i]>biggest){
+    biggest=objArr[i];
+  }
+  return biggestValue(obj,i+1,objArr,biggest);
+}
```

Comment:
In this case it would look prettier if you write it out like:

```
if (i == objArr.length) return biggest;
if (objArr[i] > biggest) biggest = objArr[i];
return biggestValue(obj, i + 1, objArr, biggest);
```